apps/feedback/test/fake_date_time_test.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Feedback.FakeDateTimeTest do
+  @moduledoc false
+  use ExUnit.Case
+
+  describe "utc_now/1" do
+    test "can start the application" do
+      assert Feedback.FakeDateTime.utc_now() ==
+               Enum.at(Tuple.to_list(DateTime.from_unix(1_562_573_043)), 1)
```

Comment:
`elem(DateTime.from_unix(1_562_573_043), 1)` would be equivalent here; [`elem` can be used to get part of a `Tuple`](https://hexdocs.pm/elixir/1.12/Kernel.html#elem/2).

That said, this test is really just hard-coding the implementation details of `Feedback.FakeDateTime`, which is not so great. Since the module is only used for creating a mock value for testing, it's a good candidate for excluding from the code coverage calculation altogether. 

If wanting to exclude from coverage, it can be done using a [configuration file for `ExCoveralls`](https://github.com/parroty/excoveralls#coverallsjson)at the root of the relevant module - `/apps/alerts/coveralls.json` has a good example of that.

Reply:
Excluded some other mock/helper files from test coverage as well